### ISSUE TITLE
.env file with session_token required

### ISSUE THREAD

**Issue by Avataw** [redwoodjs/redwood-tutorial#55]
It's trivial but after running the installation steps in the readme e.g:
```bash
git clone https://github.com/redwoodjs/redwood-tutorial
cd redwood-tutorial
yarn install
yarn rw prisma migrate dev
yarn rw dev
```
I was not able to sign up or login - it failed with 500s 
![image](https://user-images.githubusercontent.com/60570041/168471559-554248e7-8511-46d6-a5f5-22d0fd147905.png)

The console I ran `yarn rw dev` with does show an appropriate message
```
api | 13:57:13 🚨 NoSessionSecretError dbAuth requires a SESSION_SECRET environment variable that is used to encrypt session cookies. Use `yarn rw g secret` to create one, then add to your `.env` file. DO NOT check this variable in your version control system!! ```

I would suggest one of the following options:
A) mention that the .env file with a user created session_secret is required to run it out of the box
B) properly show an error toast when the signup or login button have 500s

**Comment by Avataw** [comment]
Ah I see - the Tutorial **does mention** this exact thing. 
So I would instead suggest changing the link in the Readme to the appropriate part instead -> 

~~[tutorial itself](https://redwoodjs.com/docs/tutorial/chapter1/prerequisites)~~ -> [tutorial itself](https://redwoodjs.com/docs/tutorial/intermission#using-the-example-repo-recommended)

The new link (https://redwoodjs.com/docs/tutorial/intermission#using-the-example-repo-recommended) 
dives exactly into this issue and how to run this project.

**Comment by thedavidprice** [comment]
@Avataw Agreed! Would you mind opening a PR to resolve this with the steps you outline above?

**Comment by Avataw** [comment]
Awesome - I will create a PR :) feel free to assign me ~

**Comment by lobosan** [comment]
any updates? I just went through this issue

**Comment by cannikin** [comment]
The tutorial has been updated to include this step for quite a while now:

<img width="731" alt="image" src="https://user-images.githubusercontent.com/300/202056344-100df6f3-195c-4923-930b-1d53da060239.png">

Is there something else you think we could do to help alleviate this hiccup, @lobosan ?

**Comment by lobosan** [comment]
in the Readme.md of this repo that instruction is missing, also the packages are not up to date, and I'm seeing issues with the order of imports

**Comment by dthyresson** [comment]
Hi @lobosan could you to submit a PR to add the line to generate the secret in the README?

Perhaps as part of that you might be able to link the project and have the imports be reordered?
